packages/valory/skills/abstract_round_abci/behaviours.py

```diff
@@ -168,7 +166,8 @@ def __init__(self, **kwargs: Any) -> None:
         ] = self._get_round_to_behaviour_mapping(self.behaviours)
 
         self.current_behaviour: Optional[BaseBehaviour] = None
-        self.__termination_behaviour: Optional[BaseBehaviour] = None
+        self.termination_behaviour: Optional[BaseBehaviour] = None
```

Comment:
For clarity, we could call this `concurrent_background_behaviour`. As really it is concurrent & in the background. The termination is only in so far correct from perspective of `abstract_round_abci` that it is a one-way transition (for now). Although in the future, one could imagine that the `concurrent_background_behaviour` could itself be replaced here: https://github.com/valory-xyz/open-autonomy/pull/1471/files#diff-e0565ddcb0e5eea72afcbdb00881efedc60c4664d72af704f5d428578c68bffbR2104